public/app/core/components/ForgottenPassword/ChangePassword.tsx

```diff
@@ -19,6 +25,10 @@ interface PasswordDTO {
 
 export const ChangePassword = ({ onSubmit, onSkip, showDefaultPasswordWarning }: Props) => {
   const styles = useStyles2(getStyles);
+  const [displayValidationLabels, setDisplayValidationLabels] = useState(false);
+  const [pristine, setPristine] = useState(true);
+  const [newPassword, setNewPassword] = useState('');
```

Comment:
You don't need to do this, can get the up-to-date value from `watch` instead: 
```
const {handleSubmit, watch, ... } = useForm();
const newPassword = watch('newPassword');
```

Reply:
I have changed this as requested. I've also changed this for `public/app/features/profile/ChangePasswordForm.tsx`.